### PR TITLE
Added full process name for Windows

### PR DESCRIPTION
**Pull request by nystilaps**
Under Windows process was identified by a fixed 8 letter name.
Changed drakvuf to use 'SeAuditProcessCreationInfo.ImageFileName.Name' with fallback to previous way.

### CODE REVIEW

File: src/libdrakvuf/win-processes.c [hunk 1]
```diff
@@ -153,8 +153,74 @@ addr_t win_get_current_process(drakvuf_t drakvuf, uint64_t vcpu_id)
     return process;
 }
 
+static unicode_string_t* convert_to_utf8(unicode_string_t* us)
```
Comment (tklengyel) [inline src/libdrakvuf/win-processes.c]: You could just use `drakvuf_read_unicode`

Comment (nystilaps) [reply src/libdrakvuf/win-processes.c]: Ok, I'll use it

**Comment by nystilaps** [comment]
Fixed code duplication and style.

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please.

**Comment by nystilaps** [comment]
Proposed fix is implemented.

**Comment by disaykin** [comment]
@nystilaps Use strrchr() function instead.

**Comment by nystilaps** [comment]
@disaykin Ok, done.

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please.

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please.

**Comment by tklengyel** [comment]
Almost there, but the string it is comparing against right now still starts with '\`, `Testing 'calc.exe' against '\calc.exe'`

**Comment by nystilaps** [comment]
Quick fix.

**Comment by tklengyel** [comment]
Thanks!

**Pull request merged**